app/static/js/slcNewProposal.js

```diff
@@ -299,3 +300,18 @@ function getCourseInstructors() {
                 .map((i,el) => $(el).data('username')).get()
                 .filter(val => (val))
 }
+
+
+const textareas = document.querySelectorAll("textarea");
```

Comment:
This can be done in jQuery and for consistency, we should switch it. 
For .querySelectorAll you could use [this.](https://api.jquery.com/input-selector/)
For addEventListener you could use [this.](https://api.jquery.com/change/)